src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -190,6 +153,61 @@ protected override void EndProcessing()
             }
         }
 
+        /// <summary>
+        /// Read the file and calculate the hash.
+        /// </summary>
+        /// <param name="path">Path to file which will be hashed.</param>
+        /// <param name="hash">Will contain the hash of the file content.</param>
+        /// <returns>Boolean value indicating whether the hash calculation succeeded or failed.</returns>
+        private bool ComputeFileHash(string path, out string hash)
+        {
+            byte[] bytehash = null;
+            Stream openfilestream = null;
+
+            hash = null;
```

Comment:
If compiler does not ask about this explicit initialization please remove.

Reply:
If I remove it in a test, when I get to `return hash != null` then it says:

CS0269 Use of unassigned out parameter 'hash'

CS0177 The out parameter 'hash' must be assigned to before control leaves the current method

so I think it is needed.